### PR TITLE
ci: update codeowners for benchpress

### PR DESCRIPTION
**Pull request by devversion**
Pawel and myself have investigated benchpress for quite some time now (and made changes),
and I believe we have the most up-to-date understanding in the team- so it would make sense
to add us to the owners.

### CODE REVIEW

**Review by pkozlowski-opensource — APPROVED** [review]

File: .pullapprove.yml [hunk 1]
```diff
@@ -782,7 +782,9 @@ groups:
     reviewers:
       users:
         - alxhub
```
Comment (josephperrott) [inline .pullapprove.yml:784]:
```suggestion
```

We can remove @alxhub here, he was placed in the group to ensure we had at least two owners on each.

**Review by josephperrott — APPROVED** [review]
LGTM

File: .pullapprove.yml (hunk 1 above)
Comment (devversion) [reply .pullapprove.yml:784]: fixup! ci: update codeowners for benchpress

Address feedback

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 1221b371503532a6a7a2d2fc8ef190680c37203a.

**Pull request closed without merging**